frontend/packages/kubevirt-plugin/src/selectors/vmim/index.ts

```diff
@@ -1 +1,2 @@
+export * from './combined';
```

Comment:
Please avoid too many acronyms in module names.

I'd suggest to name this directory `selectors/vmi-migrations` (either plural or singular) instead.

"VMI" is a quite common acronym, but "VMIM" isn't.

Reply:
you are right; changed to `vmi-migration`